src/Bridge/Symfony/Routing/Router.php

```diff
@@ -30,11 +30,20 @@
         UrlGeneratorInterface::NET_PATH => RouterInterface::NETWORK_PATH,
     ];
 
+    const CONFIG_MAP = [
+        'ABS_URL' => UrlGeneratorInterface::ABS_URL,
+        'ABS_PATH' => UrlGeneratorInterface::ABS_PATH,
+        'REL_PATH' => UrlGeneratorInterface::REL_PATH,
+        'NET_PATH' => UrlGeneratorInterface::NET_PATH,
+    ];
+
     private $router;
+    private $defaultReferenceType;
 
-    public function __construct(RouterInterface $router)
+    public function __construct(RouterInterface $router, string $defaultReferenceType)
```

Comment:
It would be nice if we could avoid modifying the router. But how would you make the method signature of e.g.
`public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH) ` ?

How would we know wether to use the value from the config or the value in $referenceType ? I think it should still be possible to overwrite the configured default value on every method call. That is why I introduced this constant CONFIG.